processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -76,6 +76,14 @@ private void appendDecimalFormatter(StringBuilder sb, ConversionContext conversi
             sb.append( conversionContext.getNumberFormat() );
             sb.append( "\"" );
         }
+        if ( conversionContext.getLocale() != null ) {
+            sb.append( ", java.util.Locale.forLanguageTag( \"" );
+            sb.append( conversionContext.getLocale() );
+            sb.append( "\" )" );
+        }
+        else {
+            sb.append( ", java.util.Locale.getDefault() " );
+        }
```

Comment:
We strive to have as much readable code as possible. having a full qualified name here isn't readable 😉. Please have a look at how we are doing it in the `getFromExpression` for the `BigDecimal` import